src/frontends/onnx/tests/onnx_import.in.cpp

```diff
@@ -910,6 +910,22 @@ OPENVINO_TEST(${BACKEND_NAME}, onnx_model_reduce_log_sum) {
     test_case.run();
 }
 
+OPENVINO_TEST(${BACKEND_NAME}, onnx_model_reduce_log_sum_18) {
+    auto model = convert_model("reduce_log_sum_18.onnx");
+
+    // input data shape (1, 1, 4, 4)
+    Inputs inputs{
+        ov::test::NDArray<float, 4>({{{{1, 1, 1, 1}, {1, 1, 1, 1}, {1, 1, 1, 1}, {1, 1, 1, 1}}}}).get_vector()};
```

Comment:
For the default `axes` such input filled with ones can be ok, but for tests with `axes` provided, it would be better to fill the first input with different values.

Reply:
I also added one more test for the axes